packages/cf-util-http/src/http.js

```diff
@@ -114,12 +108,10 @@ const BODYLESS_METHODS = ['GET', 'HEAD'];
  * @see [Global Fetch on MDN](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch)
  * @see [WHATWG Fetch API Spec](https://fetch.spec.whatwg.org/)
  */
-export function request(method, url, opts, callback) {
+export function request(method, url, opts = {}, callback) {
   if (typeof opts === 'function') {
     callback = opts;
     opts = {};
-  } else if (opts == null) {
```

Comment:
Keep this, otherwise passing a `null` to opts will set opts to null instead of `{}`